vault-github/run.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+BOLD='\033[0;1m'
+VAULT_ADDR=${VAULT_ADDR}
+VERSION=${VERSION}
+AUTH_PATH=${AUTH_PATH}
+GITHUB_EXTERNAL_AUTH_ID=${GITHUB_EXTERNAL_AUTH_ID}
+
+# Check if vault is installed
+if ! command -v vault &>/dev/null; then
+    printf "$${BOLD}Installing vault CLI ...\n\n"
+    # check if wget is installed
+    if ! command -v wget &>/dev/null; then
+        printf "wget is not installed. Please install wget in your image.\n"
+        exit 1
+    fi
+    # check if unzip is installed
+    if ! command -v unzip &>/dev/null; then
+        printf "unzip is not installed. Please install unzip in your image.\n"
+        exit 1
+    fi
+    # check if VERSION is latest
+    if [ "${VERSION}" = "latest" ]; then
+        INSTALL_VERSION=$(curl -s https://releases.hashicorp.com/vault/ | grep -oP '[0-9]+\.[0-9]+\.[0-9]' | tr -d '<>' | head -1)
+    else
+        INSTALL_VERSION=$VERSION
+    fi
+
+    # download vault
+    wget -q -O vault.zip https://releases.hashicorp.com/vault/$${INSTALL_VERSION}/vault_$${INSTALL_VERSION}_linux_amd64.zip
+    unzip vault.zip
+    sudo mv vault /usr/local/bin
```

Comment:
Perhaps `sudo` isn't guaranteed? How about installing into `~/.local/bin` or `~/.local/share/vault[/bin]` instead and updating PATH?

```sh
export PATH="${HOME}/.local/share/vault/bin:${PATH}"
```

Reply:
We do not have an easy way to manipulate the path yet. This would have to be managed using a shell init files too :/